Week2/task15.js

```diff
@@ -0,0 +1,25 @@
+console.log(`More JavaScript`);
+console.log(`
+Task 15 `); // testing the equality of the same arrays
+let x = [1, 2, 3];
+let y = [1, 2, 3];
+let z = y;
+console.log('I think x == y is true');
+console.log('I think x === y is true');
+console.log('I think z == y is true');
+console.log('I think z == x is true');
+

```

Comment:
Only one of these is actually true, have you checked all of them and do you understand why only 1 is true?

Reply:
I try to write a function which shows if two arrays have the same element then the function will return to true. That is what I understood from the question. I observe that with writing the equality to the console.